WebRoot/js/zimbraMail/mail/view/ZmConvView2.js

```diff
@@ -1203,7 +1207,7 @@ ZmConvReplyView.prototype._initializeToolbar = function() {
 			buttons:			buttons,
 			overrides:			overrides,
 			posStyle:			DwtControl.STATIC_STYLE,
-			buttonClassName:	"DwtToolbarButton",
+			buttonClassName:	"ZToolbarButton",
```

Comment:
Minor, if we can make indentation of couple of lines here consistent, it will be better.

Thanks

Reply:
Tab is correct in editor

Reply:
No problem. Looks good in my editor too.